pkg/source/source.go

```diff
@@ -80,6 +84,11 @@ func (ks *Kind) Start(handler handler.EventHandler, queue workqueue.RateLimiting
 	// Lookup the Informer from the Cache and add an EventHandler which populates the Queue
 	i, err := ks.cache.GetInformer(ks.Type)
 	if err != nil {
+		switch err.(type) {
```

Comment:
no need to for the one item switch here --

```go
if kindMatchErr, ok := err.(*meta.NoKindMatchError); ok {
   log.Error(err, /* rest of message goes here */)
}
return err
```

Reply:
Thanks.